<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarsConfigWidget</class>
 <widget class="QDialog" name="StarsConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Stars Plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_viewConstellationLinesCheckbox">
     <property name="text">
      <string>Show Constellation Lines</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_viewConstellationLabelsCheckbox">
     <property name="text">
      <string>Show Constellation Labels</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_viewDsosCheckbox">
     <property name="text">
      <string>Show Deep Sky Objects</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_viewSunCheckbox">
     <property name="text">
      <string>Show Sun</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="magLimitLabel">
     <property name="text">
      <string>Magnitude Limit:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="m_magnitudeSlider">
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="constellationColorButton">
     <property name="text">
      <string>Constellation Color</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>164</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StarsConfigWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StarsConfigWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
